gulpfile.js

```diff
@@ -84,5 +94,5 @@ gulp.task('dev-add-livereload', function () {
   process.env.DEV_ADD_LIVERELOAD = true;
 });
 
-gulp.task('default', ['build', 'img']);
+gulp.task('default', ['build', 'build-page', 'img']);
```

Comment:
I don't think we want to `build-page` when we just run `gulp`. the `build-page` command just builds a demo right? It's fine to have a separate gulp task but I think that should only be called explicitly

Reply:
... Makes sense I guess... ✅